src/Npgsql/NpgsqlLargeObjectStream.cs

```diff
@@ -212,31 +168,23 @@ async Task<long> GetLength(bool async)
         /// <param name="origin">A value of type SeekOrigin indicating the reference point used to obtain the new position.</param>
         /// <returns></returns>
         public override long Seek(long offset, SeekOrigin origin)
-            => Seek(offset, origin, false).GetAwaiter().GetResult();
+            => base.Seek(offset, origin);
```

Comment:
While the comment makes sense, I would avoid overriding members only to specify a comment. The important part about a round-trip may be included into the remarks section of the class.

Reply:
We have then the same issue for all public methods of this class, should we move all methods comments into class comments ?!?